docs/performance/inserts/tuning.rst

```diff
@@ -132,9 +159,11 @@ even specific partitions) on your own schedule. If so, you can use the
 .. _fulltext indexes: https://crate.io/docs/crate/reference/en/latest/sql/fulltext.html
 .. _natural primary key: https://en.wikipedia.org/wiki/Natural_key
 .. _OPTIMIZE: https://crate.io/docs/crate/reference/en/latest/sql/reference/optimize.html
+.. _ANALYZE: https://cratedb.com/docs/crate/reference/en/latest/sql/statements/analyze.html
 .. _refresh_interval: https://crate.io/docs/crate/reference/en/latest/sql/reference/create_table.html#refresh-interval
 .. _Solid-State Drives: https://en.wikipedia.org/wiki/Solid-state_drive
 .. _surrogate primary key: https://en.wikipedia.org/wiki/Surrogate_key
 .. _system column: https://crate.io/docs/crate/reference/en/latest/sql/administration/system_columns.html
 .. _translog.durability: https://crate.io/docs/crate/reference/en/latest/sql/reference/create_table.html#translog-durability
 .. _turning column indexes off: https://crate.io/docs/crate/reference/en/latest/sql/ddl/indices_full_search.html#disable-indexing
+:: _Overload Protection: https://cratedb.com/docs/crate/reference/en/latest/config/cluster.html#overload-protection
```

Comment:
There is a better way to link to other Spinx documentation, like
```
:ref:`crate-reference:overload_protection`
```

Reply:
See https://github.com/crate/cratedb-guides/pull/6#pullrequestreview-1734102915: